guides/source/association_basics.md

```diff
@@ -956,6 +957,10 @@ end
 
 If you set the `:validate` option to `true`, then associated objects will be validated whenever you save this object. By default, this is `false`: associated objects will not be validated when this object is saved.
 
+##### `:optional`
+
+If you set the `:optional` option to `true`, then associated object will be validated for presence. By default, this is `false`: associated objects will be validated for presence.
```

Comment:
**optional:** By default, this is **false**: associated objects will **be** validated for presence.